_posts/en/newsletters/2021-12-15-newsletter.md

```diff
@@ -0,0 +1,195 @@
+---
+title: 'Bitcoin Optech Newsletter #179'
+permalink: /en/newsletters/2021/12/15/
+name: 2021-12-15-newsletter
+slug: 2021-12-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to allow relay of
+transactions with zero-value outputs in some cases and summarizes a
+discussion about preparing LN for the adoption of PTLCs.  FIXME:harding
+also included are...
+
+## News
+
+- **Adding a special exception for certain uneconomical outputs:**
+  since our description in [Newsletter #162][news162 unec], Jeremy Rubin
+  has [renewed][rubin unec] discussion on the Bitcoin-Dev mailing list
+  about allowing transactions to create outputs with values below the
+  [dust limit][topic uneconomical outputs].  The dust limit is a
+  transaction relay policy that nodes use to discourage users from
+  creating UTXOs that don't make economic sense to spend.  UTXOs need to
+  be stored by at least some full nodes until spent, and in some cases
+  be able to be retrieved quickly, so allowing *uneconomical outputs*
+  may create problems for no good reason.
+
+    However, there may be a use for zero-value outputs in [CPFP][topic
+    cpfp] fee bumping where the none of the funds from the transaction
+    being fee bumped can be spent---all the funds used for fee bumping
+    need to come from separate UTXOs, such as in [eltoo][topic eltoo].
+    Rubin Somsen also [provided][somsen unec] an example of how zero-fee
```

Comment:
Name misspelled:
```suggestion
    Ruben Somsen also [provided][somsen unec] an example of how zero-fee
```

Reply:
source: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2021-December/019637.html

Reply:
Oof, there are few things that embarrass me more than getting names wrong, thank you for catching before publication!